pkg/fileutils/fileutils_test.go

```diff
@@ -384,73 +376,82 @@ func TestMatches(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		res, _ := regexpMatch(test.pattern, test.text)
+		pm, err := NewPatternMatcher([]string{test.pattern})
+		if err != nil {
+			t.Fatalf("invalid pattern %s", test.pattern)
```

Comment:
minor thing but can you use %q instead so its quoted. Otherwise they could get something like:
`invalid pattern john`  if the pattern just happened to be a word that fits nicely into this sentence.